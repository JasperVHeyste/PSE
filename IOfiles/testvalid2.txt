device
Office_Printer5
3
40

job
1
1
SergeDemeyer

