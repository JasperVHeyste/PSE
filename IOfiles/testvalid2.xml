<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<DEVICE>
		<name>Office_Printer5</name>
		<emissions>3</emissions>
		<speed>40</speed>
	</DEVICE>
	<JOB>
		<jobNumber>1</jobNumber>
		<pageCount>1</pageCount>
		<userName>SergeDemeyer</userName>
	</JOB>
</SYSTEM>
