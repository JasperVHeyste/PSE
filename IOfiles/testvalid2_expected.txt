device
Office_Printer5
3
40

job
1
1
SergeDemeyer

compensation
Students 4 Climate
1234

